src/oscar/apps/dashboard/users/views.py

```diff
@@ -82,12 +82,11 @@ def apply_search_filters(self, queryset, data):
             # If the value is two words, then assume they are first name and
             # last name
             parts = data['name'].split()
-            if len(parts) == 2:
-                condition = Q(first_name__istartswith=parts[0]) \
-                    | Q(last_name__istartswith=parts[1])
-            else:
-                condition = Q(first_name__istartswith=data['name']) \
-                    | Q(last_name__istartswith=data['name'])
+            # always true filter
+            condition = Q(pk__isnull=False)
```

Comment:
Set the initial filter condition to True(like an empty filter), then we can use something like “condition &= new_condition” in a for loop to combine more conditions(something like a>1 and b<2 and ...)